buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -26,7 +26,7 @@ fun Project.configureIntellijPlugin() {
     instrumentCode.set(false)
     pluginName.set("git-machete-intellij-plugin")
     version.set(intellijVersions.buildTarget)
-    plugins.set(listOf("Git4Idea")) // Needed solely for ArchUnit
+    plugins.set(listOf("Git4Idea", "org.jetbrains.plugins.github")) // Needed solely for ArchUnit
```

Comment:
Probably here it won't even be needed (if `./gradlew build` passes without `org.jetbrains.plugins.github` here, then it's okay)